### ISSUE TITLE
[Inference Engine initialization error]

### ISSUE THREAD

**Issue by sameerthahir** [openvinotoolkit/openvino#6368]
##### System information (version)
- Hardware Raspberry Pi 3B(1GB RAM - 128MB shared with video) ,PiCam enabled
- Intel Myriad VPU [Movidius Neural Compute Stick](v1 )
- OpenVINO => 2020.1
- Host Operating System / Platform => Raspbian GNU/Linux 10 (buster) 5.10.17-v7+
- Container Operating System / Platform => Raspbian GNU/Linux 10 (buster) 5.10.17-v7+
- Interpreter => Python 3.7
- Docker version 20.10.7, build f0df350
- Problem classification: Inference Engine Initialization error

- Model name: Mobilenet SSD

##### Detailed description
I am trying to run an object detection algorithm using the [Mobilenet SSD model](https://www.pyimagesearch.com/2019/04/08/openvino-opencv-and-movidius-ncs-on-the-raspberry-pi/) on my Raspberry Pi 3B with a Picam and Intel Neural Stick v1. I have been able to successfully run this on my host OS but I have ran into an error while running this in the container.

##### Steps to reproduce

- Raspberry Pi 3B connected with a Picam and Intel Neural Stick v1 plugged into USB.
- Ran Dockerfile with following steps starting from raspbian buster armv7l base
- Downloaded[ openvino runtime toolkit ](https://download.01.org/opencv/2020/openvinotoolkit/2020.1/l_openvino_toolkit_runtime_raspbian_p_2020.1.023.tgz)and extracted to openvino folder
- Sourced ~/openvino/bin/setupvars.sh
- Executed ~/openvino/install_dependencies/install_NCS_rules.sh
- Performed [additional steps](https://github.com/openvinotoolkit/dlstreamer_gst/issues/54#issuecomment-608529488)
- Replugged NCS 
- Ran docker run -it --privileged=true -v /opt/vc:/opt/vc --env LD_LIBRARY_PATH=/opt/vc/lib  --device /dev/vchiq:/dev/vchiq   -v /dev:/dev c2e42f  /bin/bash
- These are for the rpi libraries are accessing the picam.
-v /opt/vc:/opt/vc --env LD_LIBRARY_PATH=/opt/vc/lib  --device /dev/vchiq:/dev/vchiq 

- Executed following and received the corresponding error.

**root@66e658eb10be:/realtime# python3 openvino_real_time_object_detection.py --model MobileNetSSD_deploy.caffemodel --proto MobileNetSSD_deploy.prototxt.txt -u 1
[INFO] loading model...
[INFO] starting video stream...
E: [ncAPI] [    177437] [python3] ncDeviceOpen:1012 Failed to find booted device after boot
Traceback (most recent call last):
  File "openvino_real_time_object_detection.py", line 50, in <module>
    detections = net.forward()
cv2.error: OpenCV(4.2.0-openvino) ../opencv/modules/dnn/src/op_inf_engine.cpp:831: error: (-2:Unspecified error) Failed to initialize Inference Engine backend (device = MYRIAD): Can not init Myriad device: NC_ERROR in function 'initPlugin'**

- Models and proto files and source code are attached
- [realtime_object_detection_openvino.zip](https://github.com/openvinotoolkit/openvino/files/6714865/realtime_object_detection_openvino.zip)

lsusb output

root@neuralpi1:/home/pi# lsusb
Bus 001 Device 007: ID 03e7:2150 Intel Myriad VPU [Movidius Neural Compute Stick]
Bus 001 Device 003: ID 0424:ec00 Standard Microsystems Corp. SMSC9512/9514 Fast Ethernet Adapter
Bus 001 Device 002: ID 0424:9514 Standard Microsystems Corp. SMC9514 Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

- tried this with different versions of openvino runtime toolkit (2019/2021). 
- Tried  raspbian buster aarch64 and  raspbian bullseye arm32 with no avail
- Additional facts: After seeing the error, running lsusb again will only show Intel against the selected device. (instead of Movidius...)
- This issue can be reproduced in the host also, but a reboot/replug will resolve the issue(need to run more tests to confirm).

##### Issue submission checklist

 - [ X] I report the issue, it's not a question
   
 - [X ] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [X ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by brmarkus** [comment]
The Movidius Myriad v1 compute stick is not supported anylonger since some releases.

I found "NOTE: With OpenVINO™ 2020.4 release, Intel® Movidius™ Neural Compute Stick is no longer supported." on "https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_supported_plugins_Supported_Devices.html".

With recent versions only the MyriadX (also called NCS2, Myriad-X v2) is supported.

**Comment by sameerthahir** [comment]
Hi Mark,

Thanks for the information. I had a note of it earlier. I have only tried the openvino toolkit v 2020.1 and v 2020.2 and also v 2019.12. It seems that 2020.3 also does not support ncs v1.

v 2020.1 works perfectly with the host OS. But gives me the error inside docker.

**Comment by jgespino** [comment]
Hi @sameerthahir 

Have you tried following this guide?
[Install the OpenVINO™ Toolkit for Raspbian* OS from a Docker* Image](https://www.intel.com/content/www/us/en/support/articles/000055220/boards-and-kits.html)

Regards,
Jesus

**Comment by sameerthahir** [comment]
> Hi @sameerthahir
> 
> Have you tried following this guide?
> [Install the OpenVINO™ Toolkit for Raspbian* OS from a Docker* Image](https://www.intel.com/content/www/us/en/support/articles/000055220/boards-and-kits.html)
> 
> Regards,
> Jesus

Hi Jesus,

Thanks for the link. No I havent looked at the link. I will try it out and let you know.

**Comment by sameerthahir** [comment]
> > Hi @sameerthahir
> > Have you tried following this guide?
> > [Install the OpenVINO™ Toolkit for Raspbian* OS from a Docker* Image](https://www.intel.com/content/www/us/en/support/articles/000055220/boards-and-kits.html)
> > Regards,
> > Jesus
> 
> Hi Jesus,
> 
> Thanks for the link. No I havent looked at the link. I will try it out and let you know.

Hi Jesus,

Kindly excuse the delay. I had tried the link and it worked successfully. Thanks a lot. 


I could only test it once. I haven't been able to use the devices due to personal commitments. I ran the object detection examples from the examples directory and the one from pyimagesearch.


I wanted to test it once more before I posted in a confirmation in order to be double sure that the issue doesn't crop up again. I think that will have to be done maybe next month.

Thanks a lot for your assistance and support. Will post more results personally.
